.gitignore

```diff
@@ -0,0 +1,2 @@
+*.pyc
+.venv
```

Comment:
If pyc is for Pycharm, what is venv being ignored for?  Just for my understanding.

Reply:
`*.pyc` contain the byte code which is executed by Python, not related to Pycharm.
`.venv` is me because I use that file to auto-activate my virtual env, probably shouldn't have pushed that as @dodecaphonix pointed out to me, but oh well.

Reply:
`.idea` would be for pycharm. `.pyc` is for ignoring compiled python files. `.venv` if for ignoring a virtual env if it exists in the project. When you use virtualenvwrapper, your venvs are stored globally and this `.venv` is not necessary. Can't hurt though.

Reply:
okok, thanks!